client/components/deposits-overview/next-deposit.tsx

```diff
@@ -110,6 +112,17 @@ const NextDepositDetails: React.FC< NextDepositProps > = ( {
 					</FlexItem>
 				</Flex>
 			</div>
+			{ /* Next Deposit Notice */ }
+			{ ! isLoading && (
+				<div className="wcpay-deposits-overview__notices">
+					<BannerNotice
```

Comment:
Do you think we should centre the vertical alignment of the `BannerNotice` icon & text with `align-items: center`?

`align-items: flex-start` (current)
![align-items-flex-start](https://user-images.githubusercontent.com/3147296/228079463-78655b9a-e983-4874-a802-e3cc52a3d80c.png)

`align-items: center`
![align-items-center](https://user-images.githubusercontent.com/3147296/228079459-bc0d07bd-68cd-4ef6-a4d7-78b3853a07d7.png)


This might be a change to `BannerNotice` rather than this parent component's CSS.

Reply:
Yep this would be my preference too but it was inconsistent with the core notice designs. Because the icon needs to remain at the top. I couldn't do an `align-items:center` otherwise it would look like this:

<img width="441" alt="Screenshot 2023-03-28 at 10 26 27 am" src="https://user-images.githubusercontent.com/8490476/228095879-5614bac5-ab0c-47bc-abbb-cd71d3844fdb.png">

Thanks to this guide https://css-tricks.com/snippets/css/a-guide-to-flexbox/ that's proving to be super helpful, I think I can get around that with a `align-self: flex-start` on the icon. 

<img width="433" alt="Screenshot 2023-03-28 at 10 30 35 am" src="https://user-images.githubusercontent.com/8490476/228096337-1145d887-99a2-4180-9d6f-9cf82ebc1dba.png">

Reply:
I'm wondering if I should apply these styles to the base element. I should right. This isn't a unique to this scenario.

Reply:
I've updated the base banner notice styles in [c0dc054](https://github.com/Automattic/woocommerce-payments/pull/5839/commits/c0dc05459c8c0527b227547c07551064d0263c87).